### PR TITLE
Feat/initialization design

### PR DESCRIPTION
**Pull request by kayiganwa04**
This is the initialization of the project and simple deisgn

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!-- Follow the Merge request sections. Remove those areas or default values that you don't need. -->
```
Comment (abelbordonado) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: Lets use the simplify version :D

````
Note: The content of the Template is commented.
See the raw file to see the comments

## Context

<!-- Why you do this change? What do yoy try to achieve? -->

<!-- Link to the task, or another PRs or important links referent to this task -->

## Test

<!-- How to set up the test -->

<!-- How to test step by step -->

## Platform

<!-- In which platforms you have test it -->

## Proof of Concept

<!-- Any screenshot or recording that shows how is working without need of running the code -->
````

File: README.md [hunk 2]
```diff
@@ -1 +1,36 @@
-# mediterranea-marine-nextjs
\ No newline at end of file
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```
Comment (abelbordonado) [inline README.md:1]: Remove the default README, add the readme sections from here that are applicable for now

https://github.com/abelbordonado/readme-and-templates/blob/main/readme/frontend-readme.md

File: next.config.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
```
Comment (abelbordonado) [inline next.config.js:2]: Do we need all this unsplahs things?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "mediterranea-marine-nextjs",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
```
Comment (abelbordonado) [inline package.json:9]: Can we configure eslint so it applies before doing a commit, and reject the commit otherwise?

File: src/app/page.tsx [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+                  <span className="flex items-center after:content-['/'] sm:after:hidden after:mx-2 after:text-gray-200 dark:after:text-gray-500">
+                    <svg className="w-3.5 h-3.5 sm:w-4 sm:h-4 mr-2.5" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 20 20">
+                      <path d="M10 .5a9.5 9.5 0 1 0 9.5 9.5A9.51 9.51 0 0 0 10 .5Zm3.707 8.207-4 4a1 1 0 0 1-1.414 0l-2-2a1 1 0 0 1 1.414-1.414L9 10.586l3.293-3.293a1 1 0 0 1 1.414 1.414Z" />
+                    </svg>
+                    Personal <span className="hidden sm:inline-flex sm:ml-2">Info</span>
+                  </span>
+                </li>
+                <li className="flex items-center">
+                  <span className="mr-2">2</span>
+                  Confirmation
```
Comment (abelbordonado) [inline src/app/page.tsx:51]: Lets set up i18n in this PR too 

https://react.i18next.com/

File: next.config.js (hunk 3 above)
Comment (kayiganwa04) [reply next.config.js:2]: yes but if we get images to use I will remove it

**Pull request closed without merging**